homeassistant/components/heatzy/translations/en.json

```
@@ -0,0 +1,22 @@
+{
+    "config": {
+        "flow_title": "{name}",
+        "step": {
+            "user": {
+                "title": "Register your heatzy account.",
```

Does this... really register an account? If not, maybe..?
```suggestion
                "title": "Enter your Heatzy account credentials.",
```